draft-ietf-cbor-7049bis.md

```diff
@@ -1911,7 +1912,7 @@ data items to be encoded without harm.
 # Converting Data between CBOR and JSON
 
 This section gives non-normative advice about converting between CBOR
-and JSON. Implementations of converters are free to use whichever
+and JSON. Implementations of converters MAY use whichever
```

Comment:
RFC 2119:
5. MAY   This word, or the adjective "OPTIONAL", mean that an item is truly optional. 
Well, OK, it is optional to use the advice here, but that is a bit of RFC 2119 abuse.

Reply:
Disagree.